app/src/main/res/layout/frag_app_start_dialog.xml

```diff
@@ -0,0 +1,89 @@
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="32dp"
+        android:text="@string/hint_username"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/text_input_username"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginTop="8dp"
+        android:ems="10"
+        android:hint="@string/hint_username"
+        android:inputType="text"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/text_title_username"
+        app:layout_constraintTop_toBottomOf="@+id/text_title_username" />
+
+    <TextView
+        android:id="@+id/text_title_password"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:text="@string/hint_password"
+        app:layout_constraintStart_toStartOf="@+id/text_title_username"
+        app:layout_constraintTop_toBottomOf="@+id/text_input_username" />
+
+    <EditText
+        android:id="@+id/text_input_password"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginTop="8dp"
+        android:ems="10"
+        android:hint="@string/hint_password"
+        android:inputType="text"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/text_title_username"
+        app:layout_constraintTop_toBottomOf="@+id/text_title_password" />
+
+    <RadioGroup
+        android:id="@+id/radio_apps_service_type_preference"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:gravity="center"
+        android:textAlignment="center"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_input_password">
+
+        <RadioButton
+            android:id="@+id/ssh_radio_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="true"
+            android:drawableEnd="@drawable/ic_console_24dp"
+            android:drawablePadding="8dp"
+            android:padding="10dp"
+            android:text="@string/client_ssh" />
+
+        <RadioButton
+            android:id="@+id/vnc_radio_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:drawableEnd="@drawable/ic_desktop_windows_black_24dp"
+            android:drawablePadding="8dp"
+            android:padding="10dp"
```

Comment:
Let's try to stick with material design increments of 4, so this should either be 8dp or 12dp.